cn/_includes/templates/concept.md

```diff
@@ -24,7 +24,7 @@
 
 {% if whatsnext %}
 
-## 开始下一步
+## What's next
```

Comment:
This is the caption for a subsection? Translating this to Chinese seems okay to me.

Reply:
yes, it is a cation. Translation is OK to me too.